tests/IResearch/IResearchQueryPhraseTest.cpp

```diff
@@ -37,7 +37,6 @@
 #include "common.h"
 #include "store/mmap_directory.hpp"
 #include "utils/index_utils.hpp"
-#include "utils/string_utils.hpp"
```

Comment:
absl::Substitute is now used by this test. Should the corresponding header file be included here?